prisma/schema.prisma

```diff
@@ -64,6 +64,7 @@ model User {
     createdAt     DateTime               @default(now()) @map("created_at")
     updatedAt     DateTime               @default(now()) @updatedAt @map("updated_at")
     featureFlags  String[]               @default([]) @map("feature_flags")
+    token         String?                @unique
```

Comment:
s/token/password_reset_token

Reply:
also add a token_expiry_time as DATETIME

Reply:
Sure. Will do that